### DISCUSSION TITLE
Updating body element

### DISCUSSION THREAD

**Discussion by nathanielnakache** [swup/swup#882] [Help] [answered]
### Describe your problem 🧐

Hello,

After several attempts, I haven't been able to find a solution. In my website, I retrieve scripts via the backend, which are then injected onto a page.

During the initial visit, the chat appears correctly. However, during navigation, the chat no longer appears. I've tried several plugins (head, script) and tags (data-ignore-script, data-swup-reload-script), but the issue persists.

Here's an example of the script generated after processing on the backend:

![Desktop-screenshot](https://github.com/swup/swup/assets/25710253/3598cba8-e8aa-4bbb-9d85-087fb79090ef)

If anyone has suggestions or insights on how to resolve this issue, I'd greatly appreciate it.

Thank you.

### What is your question? 🙋‍♀️

How can I load this snippet of code so that it will be executed in the DOM as expected?

I'd like to clarify that these are scripts loaded from the backend, inserted by the user, and then loaded and injected into the <head> section of the layout.

### URL to minimal reproduction 🔗

workinlocal

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for answers to my question
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by daun** [comment]
Hello ✌️ Can you share a link to the site in question? Or the exact markup of the site? I'm afraid this is impossible to debug without access to an actual demo.

**Reply by nathanielnakache** [reply]
Hi,

yes thank you.

https://onetimesecret.com/secret/h0h2q8o98hbf12dkxri516sgealdmv

**Comment by daun** [comment, marked as answer]
I see. Swup doesn't support swapping out the whole body. You've set `<body id="swup">`. If you pick another element down the tree (like a `main` element, i.e. your main content container), it should work as expected. If your chat widget is outside of that main content container, it will not be modified and just work as expected.

**Reply by nathanielnakache** [reply]
Just did it. It works! Spent 10 hours on it ;)

Thanks a lot!

**Reply by daun** [reply]
🧃
